### DISCUSSION TITLE
Build debian package for vim

### DISCUSSION THREAD

**Discussion by sabahsawor** [vim/vim#13436] [General]
Hi All,

Is there any instruction on how to build a .deb file for vim, I couldn't find anything like that in here.

I appreciate any help.

**Comment by chrisbra** [comment]
That is more a packaging question and not so much a Vim question. Please check with the Debian community how to package a deb file properly. You can also use [Debians Vim mirror](https://salsa.debian.org/vim-team/vim) (it contains the debian folder with all the packaging magic) and you should be able to build a proper Vim package from there. (run `debuild -b -us -uc`)
